packages/react/src/elements/components/Typography/Typography.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import "./typography.scss";
+
+export const validColors = [
```

Comment:
Are these constants? Since we're exporting them, we should use the proper naming convention and freeze them.

```js
export const VALID_COLORS = Object.freeze([ /* ... */ ]);
```